cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,25 @@
+import socket
+import time
+import os
+
+from django.core.management import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Waits for database to be ready for connection"  # noqa: VNE003
+
+    def handle(self, *args):
+        port = int(os.environ["POSTGRES_PORT"])  # 5432
```

Comment:
It's a good practice to ensure that the environment variable exists before you try to use it. Consider adding a check to see if 'POSTGRES_PORT' is in `os.environ` before trying to access it. If it's not present, you could throw an error or use a default value.